index.html

```diff
@@ -1,21 +1,142 @@
+
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="/css/main.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap"
+      rel="stylesheet"
+    />
 
     <title>Responsive grid project</title>
-</head>
+  </head>
+
+  <body>
+    <header>
+      <div class="toggle-menu">
+        <hr class="line-one" />
+        <hr class="line-two" />
+        <hr class="line-three" />
+      </div>
+
+      <div class="blue-box"></div>
+      <h1>Maya Nelson</h1>
+      <h2>Project Manager</h2>
+
+      <!-- <ul class="header-nav-menu"> -->
+      <!-- <li class="about-me"><a href=""></a>ABOUT ME</li>
+      <li><a href=""></a>RESUME</li>
+      <li><a href=""></a>PROJECTS</li>
+      <li><a href=""></a>CONTACT</li> -->
+    </header>
+
+    <main>
+      <section class="card">
+        <img
+          src="AdobeStock_230407429.webp"
+          alt="woman-picture"
+          class="profile-img"
+        />
+        <p class="name">Maya<br />Nelson</p>
+        <hr class="card-line" />
+
+        <div class="card-buttons">
+          <button class="card-buttons1">RESUME</button>
+          <button class="card-buttons2">PROJECTS</button>
+        </div>
+
+        <div class="social-bar">
+          <img
+            src="facebook-logo-icon-transparent-background-black-facebook-icon-clipart-19-removebg-preview.png"
+            alt="facebook-icon"
+            class="facebook"
+          />
+          <img
+            src="png-clipart-black-and-white-tiff-twitter-mid-autumn-logo-leaf-computer-wallpaper-removebg-preview.png"
+            alt="twitter-icon"
+            class="twitter"
+          />
+          <img
+            src="53-533439_linkedin-png-icono-linkedin-png-clipart-removebg-preview.png"
+            alt="linkedin-icon"
+            class="linkedin"
+          />
+          <img
+            src="5ecec78673e4440004f09e77-removebg-preview.png"
+            alt="instagram-icon"
+            class="instagram"
+          />
+        </div>
+      </section>
 
-<body>
+      <section class="info-paragraph">
+        <div class="paragraph-text">
+          <h3><strong>Hello</strong></h3>
+          <h4>Here's who I am & what I do</h4>
+          <p class="paragraph">
+            I'm a paragraph. Click here to add your own text and edit me. It’s
+            easy. Just click “Edit Text” or double click me to add your own
+            content and make changes to the font.
+          </p>
 
-    <!-- Add your markup here -->
+          <p class="paragraph">
+            I’m a great place for you to tell a story and let your users know a
+            little more about you.
+          </p>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="contact">
+        <p><strong>Call</strong></p>
```

Comment:
the phone number should be the element after the `Call` heading, in the HTML, so a screenreader will read them one after the other. The problem is using grid to place the elements in a square. Instead think of 'call, write, follow' as 3 wrapper elements in a single row. Then inside each wrapper, there is a heading and an element below it with the phone number / email / icons.